docs/edex/new-grid.md

```diff
@@ -15,105 +17,50 @@ If the parameters in the grib file haven't been previously specified, another ch
+Download an example grib2 file (make sure the extension is `.grib2` or the [EDEX distribution file](../data-distribution-files/#editing-an-edex-data-distribution-file) may not recognize it), and then copy to the manual ingest point `/awips2/data_store/ingest/`:
 
-        wget https://downloads.unidata.ucar.edu/awips2/current/files/14102318_nmm_d01.GrbF00600 -O wrf.grib
-    
-        cp wrf.grib /awips2/data_store/ingest/
-    
-    Remember that the data distribution file (`/awips2/edex/data/utility/common_static/base/distribution/grib.xml`) will match filenames which have the `*.grib*` extension.
+    wget https://downloads.unidata.ucar.edu/awips2/current/files/CPTI_00.50_20180502-000144.grib2 -O cpti.grib2
 
-2. Confirm that the grib file decodes in the grib log file:
-    
-        edex log grib
-    
-        INFO [Ingest.GribDecode] /awips2/data_store/ingest/grib/20141026/14/wrf.grib processed in: 0.1200 (sec) Latency: 21.8080 (sec)
-        INFO [Ingest.GribDecode] /awips2/data_store/ingest/grib/20141026/14/wrf.grib processed in: 0.1180 (sec) Latency: 21.8140 (sec)
-        INFO [Ingest.GribDecode] /awips2/data_store/ingest/grib/20141026/14/wrf.grib processed in: 0.4230 (sec) Latency: 21.8360 (sec)
-        INFO [Ingest.GribDecode] /awips2/data_store/ingest/grib/20141026/14/wrf.grib processed in: 0.2240 (sec) Latency: 21.9140 (sec)
-        
-        ...
-
-3. Check that the hdf5 data directory exists for our unnamed grid
+    cp cpti.grib2 /awips2/data_store/ingest/
 
-        ls -latr /awips2/edex/data/hdf5/grid/GribModel:7:0:89
+### Check Grib Logs
 
-    Though the grib file has been decoded, it has been given a generic name with its center, subcenter, and process IDs (7, 0, 89, respectively). 
+Confirm that the grib file decodes in the grib log file.
     
-### Grib2 Products
-
-1. Download an example grib2 file and rename to a `*.grib2` extension, then copy to the manual ingest point `/awips2/data_store/ingest/` 
+Look in the current log file (/awips2/edex/logs/edex-ingestGrib-[YYYYMMDD].log) for the following:
 
-        wget https://downloads.unidata.ucar.edu/awips2/current/files/CPTI_00.50_20180502-000144.grib2 -O cpti.grib2
+    INFO [Ingest.GribDecode] /awips2/data_store/ingest/cpti.grib2 processed in: 0.1200 (sec) Latency: 21.8080 (sec)
+    INFO [Ingest.GribDecode] /awips2/data_store/ingest/cpti.grib2 processed in: 0.1180 (sec) Latency: 21.8140 (sec)
+    INFO [Ingest.GribDecode] /awips2/data_store/ingest/cpti.grib2 processed in: 0.4230 (sec) Latency: 21.8360 (sec)
+    INFO [Ingest.GribDecode] /awips2/data_store/ingest/cpti.grib2 processed in: 0.2240 (sec) Latency: 21.9140 (sec)
     
-        cp cpti.grib2 /awips2/data_store/ingest/
+    ...
     
-    Remember that the data distribution file (`/awips2/edex/data/utility/common_static/base/distribution/grib.xml`) will match filenames which have the `*.grib*` extension.
+!!! warning "This step may fail if the parameter is not yet defined.  The error will look like:"
```

Comment:
Specifically for the cpti.grib2 file - the parameters don't exist so you do need to create the tables first before you can actually see it ingested. I know you have the note in here, but maybe explicitly state in this specific case, we have to define the parameters first.

Reply:
I think the idea (similar to what Jason Burke's does at times) is to have the user purposely encounter the error, so they're familiar with it.

How is this:
![image](https://user-images.githubusercontent.com/21114460/162518662-70d6c844-486a-4099-928e-b2a89679639a.png)

Reply:
Agreed. 

The only other thing to maybe mention, is in the second code block, it shows that it was "ingested" (line 2) but it really wasn't.